read.version.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Reverted this [change from the last PR](https://github.com/coveo/search-ui/pull/1665/files#diff-0b600ebb8ad7dde0759d71aaa725c2e508f9c53c794a048d802019cb7c74a07bR1) to stay consistent with other bash files in the repo since it did not do anything.